### ISSUE TITLE
Parsing options argument default by parseWhole

### ISSUE THREAD

**Issue by romanrostislavovich** [linterhub/usage-parser#148]
# Expected Behavior

Parsing CLI arguments by default without any sections names, it's property `parseWhole`

## Actual Behavior

Some CLI have the following doc:
```
Options: 

-p, --prop    Description

Another sections: 

-c, --config <path>   Description 
```

For parse CLI of such, we need to set custom configuration. I suggest changing the default behavior for parsing all CLI doc like a `parseWhole` property.

**Comment by romanrostislavovich** [comment]
Part of #170

**Comment by romanrostislavovich** [comment]
Closed in #174
